doc/company.texi

```diff
@@ -1266,9 +1270,9 @@ For more details on @acronym{CAPF}, @ref{Completion in Buffers,,,elisp}.
 @end defun
 
 @defun company-dabbrev-code
-This backend works similarly to the Emacs built-in package
-@emph{dabbrev}, parsing completion candidates from the text in open
-buffer(s).  Internally, its based on the backend
+This backend works similarly to the built-in Emacs package
+@emph{dabbrev}, searching for completion candidates in the text
+available in the open buffer(s).  Internally, its based on the backend
```

Comment:
Suggestion:

> searching for completion candidates inside the contents of all open buffers. Or a subset of them.

Reply:
Updated to "inside the contents of the open buffer(s)", because I have a mild preference to give as an overall definition here as possible.